### ISSUE TITLE
where is the code of 'multi-scale testing'

### ISSUE THREAD

**Issue by sumaliqinghua** [HRNet/HRNet-Human-Pose-Estimation#1]
Excuse me,
thanks for your great work, I noticed there is 'we also perform six-scale pyramid testing procedure (multi-scale testing)' in your paper, and you perform some result about it in Appendix. But I can't find it in this repository, is anything I ignore? or could you tell me how to realize it by this repository. Thanks again for your hard work

**Comment by MaxChu719** [comment]
I also want to know the multi-scale code as i cannot reproduce the MPII test set result using the pre-trained model. I even email the author but I got no response...

**Comment by MaxChu719** [comment]
#5 
@sumaliqinghua I have posted an issue with my implementation of the multi-scale testing code. However, I cannot reproduce the MPII test result. Maybe my code will help you in some ways. If you can slove the reproduction problem, please let me know. :)
